bundles/org.openhab.io.openhabcloud/README.md

```diff
@@ -141,6 +147,15 @@ The additional parameter for these variants have the following meaning:
 
 These parameters may be skipped by setting them to `null`.
 
+### Hide Notification Actions
```

Comment:
This section should go right before the examples and AFTER the action syntax section.